pkg/driver/command_driver.go

```diff
@@ -71,7 +72,26 @@ func (d *CommandDriver) exec(op *driver.Operation) error {
 	}
 	cmd.Env = pairs
 	cmd.Stdin = bytes.NewBuffer(data)
-	out, err := cmd.CombinedOutput()
-	fmt.Fprintln(op.Out, string(out))
-	return err
+
+	// Make stdout and stderr from driver available immediately
+
+	stdout, _ := cmd.StdoutPipe()
+	go func() {
+		io.Copy(op.Out, stdout)
+	}()
+	stderr, _ := cmd.StderrPipe()
+	go func() {
+		io.Copy(op.Out, stderr)
+	}()
+	err = cmd.Start()
+	if err != nil {
+		return fmt.Errorf("cmd.Start(%s) failed: %v", d.cliName(), err)
```

Comment:
nit: the error message will make more sense to the user if it is phrased in terms of externals the user will understand rather than internal code details. So something like "start of command driver %s failed: %v", although I don't understand the context well enough to provide a good message.